tests/test_build_latex.py

```diff
@@ -814,3 +815,106 @@ def test_maxlistdepth_at_ten(app, status, warning):
     print(status.getvalue())
     print(warning.getvalue())
     compile_latex_document(app)
+
+
+@pytest.mark.skipif(docutils.__version_info__ < (0, 13),
+                    reason='docutils-0.13 or above is required')
+@pytest.mark.sphinx('latex', testroot='latex-table')
+def test_latex_table(app, status, warning):
+    app.builder.build_all()
+    result = (app.outdir / 'test.tex').text(encoding='utf8')
+    tables = {}
+    for chap in re.split(r'\\chapter(?={.*})', result)[1:]:
+        sectname, content = chap.split('}', 1)
+        tables[sectname[1:]] = content.strip()
+
+    # simple_table
+    table = tables['simple table']
+    assert ('\\noindent\\begin{tabulary}{\\linewidth}{|L|L|}' in table)
+    assert ('\\hline\n'
+            '\\sphinxstylethead{\\relax \nheader1\n\\unskip}\\relax &'
+            '\\sphinxstylethead{\\relax \nheader2\n\\unskip}\\relax' in table)
+    assert ('\\hline\ncell1-1\n&\ncell1-2\n\\\\' in table)
+    assert ('\\hline\ncell2-1\n&\ncell2-2\n\\\\' in table)
+    assert ('\\hline\ncell3-1\n&\ncell3-2\n\\\\' in table)
+    assert ('\\hline\\end{tabulary}' in table)
+
+    # table having :widths: option
+    table = tables['table having :widths: option']
+    assert ('\\noindent\\begin{tabulary}{\\linewidth}'
```

Comment:
should be ``assert ('\\noindent\\begin{tabular}'`` after remark at https://github.com/sphinx-doc/sphinx/pull/3389/files#r99466224

and ``\end{tabular}`` of course